README.rst

```diff
@@ -0,0 +1,98 @@
+.. role:: bash(code)
+   :language: bash
+
+=========
+pyUmbral
+=========
+
+.. image:: https://travis-ci.org/nucypher/pyUmbral.svg?branch=master
+    :target: https://travis-ci.org/nucypher/pyUmbral
+
+pyUmbral is a 100% python implementation of Umbral using OpenSSL and Cryptography.io,
+enabling users to perform *split-key proxy-rencryption* and public key encryption
+in an understandable and usable manner.
+
+**Alice encrypts data for herself**
+
+.. code-block:: python
+
+    from umbral import umbral, keys
+
+    # Generate a key pair for Alice
+    alices_private_key = keys.UmbralPrivateKey.gen_key()
+    alices_public_key = private_key.get_pubkey()
+
+    # Encrypt data with bob's public key
+    plaintext = b'Proxy Re-encryption is cool!'
+    ciphertext, capsule = umbral.encrypt(alices_public_key, plaintext)
+
```

Comment:
Comment should say "Encrypt data with Alice's public key"

Reply:
:heavy_check_mark: